zproject/urls.py

```diff
@@ -196,6 +196,10 @@
     url(r'^users/me/enter-sends$', 'zerver.lib.rest.rest_dispatch',
         {'POST': 'zerver.views.user_settings.change_enter_sends'}),
 
+    # users/settings/name/update -> zerver.views.user_settings
+    url(r'^users/settings/name/update$', 'zerver.lib.rest.rest_dispatch',
+        {'POST': 'zerver.views.user_settings.json_change_user_full_name'}),
```

Comment:
I think the proper REST style for this would be to use the existing endpoint style for modifying another user:
```
     url(r'^users/(?P<email>(?!me)[^/]*)$', 'zerver.lib.rest.rest_dispatch',
         {'PATCH': 'zerver.views.users.update_user_backend',
          'DELETE': 'zerver.views.users.deactivate_user_backend'}),
```

It might even make sense to just modify that endpoint to support changing the name as well rather than adding a new endpoint...